07-advanced-git/07-01-git-log/README.md

```diff
@@ -0,0 +1,116 @@
+Looking at the man page for this command:  
+```bash
+Documentation:
+--oneline
+This is a shorthand for "--pretty=oneline --abbrev-commit" used together.
+--abbrev-commit
+Instead of showing the full 40-byte hexadecimal commit object name, show only a partial prefix. Non default number of digits can be specified with "--abbrev=<n>" (which also modifies diff output, if it is displayed).
+```
+Thus when running the command.
+```bash
+git log --oneline
+```
+![Git log with only oneline](res/small_picture_2.png)
+
+And boom, this was what we wanted at the start. Now we can see that we have added the landing page and also changed the extension of the landing page.  
+
+But there is something weird here...  
+We can see that there is a commit tagged with origin/master, along with the commits of our current branch, frontend/landing. As we have established previously in chapter 6, all branches start off somewhere. In this example, the branch frontend/landing branches out from the origin/master branch. As such, git log makes a reference to this.
+
+Luckily, we don't have to guess as there is a flag which will show us the relationship between these commits.  
+
+#### graph
+To make git log show the relationships between commits, we just have to add the `--graph` flag. This flag shows a 'graph' of the commits, which for us means the relationship the commits have.  
+Combining this with `--oneline` flag we showed above, makes for a very clean and concise git log.  
+
+(Try this out yourself and see what happens!)
+
+```bash
+git log --oneline --graph
+```
+![Git log with oneline and graph](res/small_picture_3.png)
+
+## Looking at the big picture
+Now that you know how to look at commits tagged to your branch, you might also want to see how the entire repository might look like.  
+Luckily we have a flag to look at the log of the entire repository.  
+
+#### all
+The `--all` flag shows the `git log`; including all the commits from every branch you have in the repository.  
+We will also be combining this flag with all previous flags to get the best overview of a project. (In our opinion)
+```bash
+git log --oneline --graph --all
+```
+![Git log with oneline graph and all](res/big_picture_1.png)
+
+Now we can see that our frontend landing page feature has already been merged down to our local master branch. We can also see that after `git fetch` (To fetch any changes from GitHub), teammate B had already made more commits on his feature branch and the local `backend/start-server` branch is behind.  
+
+## Wrapping up
+To end off, the `git log` command actually has a lot more features and flags which are a bit more nuanced and situational. This chapter demonstrates the most commonly used features and flags which we hope will be usefully to you.  
+
+As mentioned above, below are additional resources:  
+[More features of git log](https://www.atlassian.com/git/tutorials/git-log)
```

Comment:
```suggestion
- [More features of git log](https://www.atlassian.com/git/tutorials/git-log)
```

Reply:
To clear up the previous confusion, I meant making these a list of items instead